docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -776,6 +776,22 @@ Here's a JSON sample that the preceding code can read. The resulting summary mes
 
 [!code-json[](~/samples/snippets/core/system-text-json/csharp/Universities.json)]
 
+### Read from a stream using Utf8JsonReader
+
+When reading a large file (a gigabyte or more in size, for example), you might want to avoid having to load the entire file into memory at once. For this scenario, you can use a <xref:System.IO.FileStream>.
+
+When using the `Utf8JsonReader` to read from a stream, the following rules apply:
+
+* The buffer containing the partial JSON payload must be at least as big as the largest JSON token within it so that the reader can make forward progress.
+* The buffer must be at least as big as the largest sequence of white space within the JSON.
+* The reader doesn't keep track of the data it has read until it completely reads the next <xref:System.Text.Json.Utf8JsonReader.TokenType%2A> in the JSON payload. So when there are bytes left over in the buffer, you have to pass them to the reader again. You can use <xref:System.Text.Json.Utf8JsonReader.BytesConsumed%2A> to determine how many bytes are left over.
+
+The following code illustrates how to read from a stream. The example shows a <xref:System.IO.MemoryStream>, but similar code will work with a <xref:System.IO.FileStream>. The code starts with a 4KB buffer and doubles the buffer size each time it finds that the size is not big enough to fit a complete JSON token, which is required for the reader to make forward progress on the JSON payload. The JSON sample provided in the snippet triggers a buffer size increase only if you set a very small initial buffer size, for example, 10 bytes.
```

Comment:
The code currently doesn't start with a 4KB buffer, so currently this comment is a bit confusing. I can provide a re-wording suggestion later.

Reply:
One of my code changes was to change the buffer definition to 4096, but I changed it back to 10 for testing and apparently neglected to restore 4096.